index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <a class="logo" href="#">My Logo</a>
+            
+            <img src="statics/images/Union.png" alt="Menu">
+               
+        </nav>
+        <section>
+           <h1>I’m Taher
+            Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div>
+                   <a class="icon" href="#"><img src="statics/images/twitter.png" alt="twitter link"></a>
+                   <a class="icon" href="#"><img src="statics/images/Linkedin_icon.png" alt="linked link"></a>
+                   <a class="icon" href="#"><img src="statics/images/M.png" alt="M"></a>
+                   <a class="icon" href="#"><img src="statics/images/github.png" alt="github link"></a>
+                   <a class="icon" href="#"><img src="statics/images/doits.png" alt="doit"></a>
```

Comment:
- [ ] Well done on wrapping the icons inside the `<a>` tags. However, also wrap them inside the `<ul>` and `<li>` tags.

Reply:
Could you tell me the utilities to use such wrapping structure in this case.